### DISCUSSION TITLE
Ability to get assistant already created

### DISCUSSION THREAD

**Discussion by brandonhenry** [metaskills/experts#33] [Ideas]
I see only a way to create assistants? Am I missing something or is there currently no way to get already created assistants 😬

**Comment by metaskills** [comment]
Yup, since we default to OpenAI's constructors, you can use the `id` property in the config to link to an existing assistant.

**Reply by brandonhenry** [reply]
Thanks a ton!! gonna give it a shot - was a bit worried about all the assistants being created 😆

**Reply by brandonhenry** [reply]
@metaskills I guess, the main readme should at least be updated with instructions on that when you have the chance :)

**Reply by metaskills** [reply]
Yup, the deployment story is not great here. It is not like OpenAI has CloudFormation. FWIW, you likely found this already in the deployment section. Adding it here for others too. https://github.com/metaskills/experts?tab=readme-ov-file#deployments
